rubin_sim/maf/metrics/incremental_template_metric.py

```diff
@@ -102,7 +111,13 @@ def run(self, data_slice, slice_point=None):
         idx_template_inputs = ok_template_input[: self.n_images_in_template]  # Images included in template
 
         Night_template_created = data_slice[self.night_col][idx_template_created]
-        N_nights_without_template = Night_template_created - data_slice[self.night_col][0]
+        N_nights_without_template = Night_template_created
+
+        Frac_into_season_template_created = calc_season(
+            np.degrees(slice_point["ra"]),
+            data_slice[self.mjd_col][idx_template_created],
+            mjd_start=self.mjd_start,
+        )
```

Comment:
How is this the fraction of the season, if its' just calc_season? (calc_season will return the integer + fractional season)

Reply:
I suppose it's only reported for the first year anyway, so it'll be fine.